src/shared/components/common/vote-buttons.tsx

```diff
@@ -0,0 +1,180 @@
+import { showScores } from "@utils/app";
+import { numToSI } from "@utils/helpers";
+import classNames from "classnames";
+import { Component, linkEvent } from "inferno";
+import { CommentAggregates, PostAggregates } from "lemmy-js-client";
+import { I18NextService } from "../../services";
+import { Icon, Spinner } from "../common/icon";
+import { PostListing } from "../post/post-listing";
+
+interface VoteButtonsProps {
+  postListing: PostListing;
```

Comment:
This needs to be made generic enough to work for both post and comments, so you'll need to expand out the counts, and my_vote into distinct fields.